de.dlr.sc.virsat.graphiti.test/src/de/dlr/sc/virsat/graphiti/diagram/BeanIndependenceSolverTest.java

```diff
@@ -139,11 +130,11 @@ public void testSolveIndependence() throws CoreException, IOException {
 		editingDomain.getVirSatCommandStack().execute(new RecordingCommand(editingDomain) {
 			@Override
 			protected void doExecute() {
-				resSet.getAndAddStructuralElementInstanceResource(sei2);
+				editingDomain.getResourceSet().getAndAddStructuralElementInstanceResource(sei2);

```

Comment:
Why is this change now needed?

Reply:
To use the resourceSet from AConceptProjectTestCase

Reply:
Maybe the variable "rs" from AProjectTestCase can cover this.